contracts/contracts/strategies/balancer/BaseBalancerStrategy.sol

```diff
@@ -431,50 +431,50 @@ abstract contract BaseBalancerStrategy is InitializableAbstractStrategy {
     }
 
     /**
-     * @notice Sets max withdrawal slippage that is considered when removing
+     * @notice Sets max withdrawal deviation that is considered when removing
      * liquidity from Balancer pools.
-     * @param _maxWithdrawalSlippage Max withdrawal slippage denominated in
+     * @param _maxWithdrawalDeviation Max withdrawal deviation denominated in
      *        wad (number with 18 decimals): 1e18 == 100%, 1e16 == 1%
      *
-     * IMPORTANT Minimum maxWithdrawalSlippage should actually be 0.1% (1e15)
-     * for production usage. Contract allows as low value as 0% for confirming
-     * correct behavior in test suite.
+     * IMPORTANT Minimum maxWithdrawalDeviation will be 1% (1e16) for production
+     * usage. Vault value checker in combination with checkBalance will
+     * catch any unexpected manipulation.
      */
-    function setMaxWithdrawalSlippage(uint256 _maxWithdrawalSlippage)
+    function setMaxWithdrawalDeviation(uint256 _maxWithdrawalDeviation)
         external
         onlyVaultOrGovernorOrStrategist
     {
         require(
-            _maxWithdrawalSlippage <= 1e18,
-            "Max withdrawal slippage needs to be between 0% - 100%"
+            _maxWithdrawalDeviation <= 1e18,
+            "Withdrawal dev. out of bounds"
         );
-        emit MaxWithdrawalSlippageUpdated(
-            maxWithdrawalSlippage,
-            _maxWithdrawalSlippage
+        emit MaxWithdrawalDeviationUpdated(
+            maxWithdrawalDeviation,
+            _maxWithdrawalDeviation
         );
-        maxWithdrawalSlippage = _maxWithdrawalSlippage;
+        maxWithdrawalDeviation = _maxWithdrawalDeviation;
     }
 
     /**
-     * @notice Sets max deposit slippage that is considered when adding
+     * @notice Sets max deposit deviation that is considered when adding
      * liquidity to Balancer pools.
-     * @param _maxDepositSlippage Max deposit slippage denominated in
+     * @param _maxDepositDeviation Max deposit deviation denominated in
      *        wad (number with 18 decimals): 1e18 == 100%, 1e16 == 1%
      *
-     * IMPORTANT Minimum maxDepositSlippage should actually be 0.1% (1e15)
-     * for production usage. Contract allows as low value as 0% for confirming
-     * correct behavior in test suite.
+     * IMPORTANT Minimum maxDepositDeviation will be 1% (1e16) for production
```

Comment:
*"will default to" (since we could change it later)

Reply:
thanks fixed